gettext-rs/src/lib.rs

```diff
@@ -87,7 +87,7 @@ use std::ffi::CString;
 use std::os::raw::c_ulong;
 use std::path::PathBuf;
 
-mod macros;
+pub mod macros;
```

Comment:
IMHO it's useful for doing something like `use gettextrs::macros::*` when one only needs the macros and none of the other stuff

Reply:
Ah, makes sense, thank you!

Reply:
While working on #53 I noticed that the `macros` module doesn't actually contain anything. Sure enough, it turned out that `#[macro_export]` moves the macros to the root of the crate. Rust 2018 doesn't affect that: the following code does not compile because it can't find the macro:

```rust
use gettextrs::{textdomain, macros::*};

fn main() {
    textdomain("hellorust").unwrap();
    printfln!("{}", gettext!("hello {}", 1));
}
```

So I made the `mod macros` private again in #54.

Reply:
Oh yes, I only tested this with Rust 2018, if it breaks stuff making it private again makes sense

Reply:
Wait, you mean with Rust 2018 you managed to somehow import *just* the macros using `use gettextrs::macros::*`? This shouldn't work!

Making `macros` public doesn't break anything, it just doesn't do anything useful either, since the module is empty.